### ISSUE TITLE
multi-line input passed to `-c` is not broken up

### ISSUE THREAD

**Issue by bitquence** [vincent-lafouasse/minishell#77]

**Comment by bitquence** [comment]
seems to be broken up by the lexer, which interprets the parts as their own words. although the expected behaviour would be to break it up into different commands, but since we aren't meant to handle more than one command in one line (except in conditionals) then we can just truncate

```
bess-f4r2s3% STR=$'echo\nhello\nworld\n\n'
bess-f4r2s3% ./minishell -c "$STR"        
hello world

```

**Comment by bitquence** [comment]
fixed in 7146b6b
